deploy/cluster_role_binding.yaml

```diff
@@ -1,11 +1,11 @@
 apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRoleBinding
 metadata:
-  name: submariner-operator:lighthouse
+  name: submariner-operator
```

Comment:
Should we not change the corresponding references in 
pkg/subctl/operator/common/embeddedyamls/yamls.go ?

Reply:
Those are already correct. These are legacy bits which I don't think we're using anymore.

Reply:
Ok, I'm not very familiar with this code. So I'll let @tpantelis to review this.

Reply:
Yes, this is only changing the name, since now those are not operator lighthouse specific permissions, just general operator ones.

This file controls the cluster permissions that submariner-operator has in general for accessing the cluster, before we only needed that for the dns configmap, etc... but now we need more permissions, and it's not lighthouse specific anymore.